pkg/kube/kube_handler_test.go

```diff
@@ -909,6 +910,45 @@ func TestGetNamespaces(t *testing.T) {
 			expectedNamespaces: []string{"bar"},
 			allowed:            true,
 		},
+		{
```

Comment:
Could you add a test where the regex value is `''`?

Reply:
I think you are looking for those 2 tests: https://github.com/kubeapps/kubeapps/pull/2671/files#diff-9a009d80344fc69fac8f2a1d383b6fa3545305bb48ac693d1b894f94ed37853aR305

Reply:
Right, sorry - these ones here are more integration tests for the GetNamespaces handler rather than unit tests for the underlying `GetNamespaces` function. Thanks.